boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -333,16 +333,34 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
             break;
         }
 
-        if (((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributes.Enabled_Reserved == 0)
+        if (((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributes.Enabled_Reserved != 1)
```

Comment:
Please keep the previous. != 1 should never be used, and this doesn't make it anymore readable.

Reply:
Reverted.